config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -391,3 +391,46 @@ periodics:
     testgrid-tab-name: aks-engine-azure-windows-1903-master
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs SIG-Windows release tests on K8s clusters provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud"
+- interval: 10h
+  name: ci-kubernetes-e2e-aks-engine-azure-master-windows-1909
+  labels:
+    preset-service-account: "true"
+    preset-azure-aksengine: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20191231-77c7890-master
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--root=/go/src"
+      - "--repo=k8s.io/kubernetes=master"
+      - "--upload=gs://kubernetes-jenkins/logs/"
+      - "--timeout=650"
+      - "--scenario=kubernetes_e2e"
+      - --
+      - "--test=true"
+      - "--up=true"
+      - "--down=true"
+      - "--deployment=aksengine"
+      - "--provider=skeleton"
+      - "--build=host-go"
+      - "--aksengine-admin-username=azureuser"
+      - "--aksengine-admin-password=AdminPassw0rd"
+      - "--aksengine-creds=$AZURE_CREDENTIALS"
+      - "--aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz"
+      - "--aksengine-public-key=$AZURE_SSH_PUBLIC_KEY_FILE"
+      - "--aksengine-winZipBuildScript=$WIN_BUILD"
+      - "--aksengine-template-url=https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/job-templates/kubernetes_1909_master.json"
+      - "--aksengine-win-binaries"
+      - "--aksengine-hyperkube"
```

Comment:
We've deprecated `--aksengine-hyperkube` flag in https://github.com/kubernetes/test-infra/pull/15844. `--aksengine-deploy-custom-k8s` should be used since aks-engine does not use hyperkube to create 1.17+ clusters anymore.

Related docs: 
- https://github.com/Azure/aks-engine/blob/master/docs/topics/kubernetes-developers.md#kubernetes-117
- https://gist.github.com/chewong/d39e40bf2dab184c650abac2139c8b23